site/source/docs/tools_reference/emcc.rst

```diff
@@ -455,6 +455,14 @@ Options that are modified or new in *emcc* are listed below:
   By default this will also clear any download ports since the ports directory
   is usually within the cache directory.
 
+.. _emcc-use-port:
+
+``--use-port=<port>``
+  [link]
```

Comment:
Actually the flag can/is also needed at compile time because it makes available the headers from the port.

Reply:
Ok